tests/unit_tests/test_merge_cubes.py

```diff
@@ -0,0 +1,105 @@
+import json
+
+import pytest
+
+from tests.utils import load_process_code, run_process_with_datacube
+
+
+@pytest.fixture
+def merge_cubes_process_code():
+    return load_process_code("merge_cubes")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [
+        (
+            {
+                'cube1': {'B01':[1,2,3],'B02':[4,5,6],'B03':[7,8,9]},
+                'cube2': {'B01':[11,12,13],'B02':[14,15,16],'B03':[17,18,19]}
```

Comment:
I handled examples 1, 2 and the one that you suggested above. But I suspect since this is a rather difficult function to implement, I did not cover all the cases that should be covered. My suggestion is to merge this PR since there is quite a few additional tests/processes which are implemented correctly and continue to work on this function in a separate PR.

Reply:
Yes. I haven't checked the code closely, but we can do that.